pages/development-resources/gccr.pl

```diff
@@ -248,7 +248,7 @@ ($$)
 			if($save_raw) {
 				$data{'line'}[$file_line_num]{'raw'} = $raw;
 			}
-		}elsif($line =~ /^\s+(\d+):\s*(\d+):(.*)/) {
+		}elsif($line =~ /^\s*(\d+):\s*(\d+):(.*)/) {
```

Comment:
yep, that's definitely the culprit! gcov is producing a line with no leading space. They should patch this upstream.... if there is an upstream